blablahelp-backend/lombok.config

```diff
@@ -0,0 +1 @@
+lombok.addLombokGeneratedAnnotation=true
```

Comment:
Die Datei muss ins Root-Verzeichnis

Reply:
moved to main project folder, outside backend